contrib/helm/harbor/values.yaml

```diff
@@ -26,10 +26,10 @@
 #       cpu: 100m
 
 persistence:
-  enabled: false
+  enabled: true
```

Comment:
If this enabled by default, users need to config all the storageClass sections. How about keep it as false by default?

Reply:
I had to set it to true to pass the test when using local storage:
https://github.com/vmware/harbor/blob/master/contrib/helm/harbor/templates/registry/registry-ss.yaml#L55

Reply:
Do you mean if set to false the chart deployment will fail ?

Reply:
Correct.